pcsx2/GS/Renderers/HW/GSRendererHW.cpp

```diff
@@ -5709,7 +5710,7 @@ GSRendererHW::CLUTDrawTestResult GSRendererHW::PossibleCLUTDraw()
 	if (m_process_texture)
 	{
 		// If we're using a texture to draw our CLUT/whatever, we need the GPU to write back dirty data we need.
-		const GSVector4i r = GetTextureMinMax(m_cached_ctx.TEX0, m_cached_ctx.CLAMP, m_vt.IsLinear(), false).coverage;
+		const GSVector4i r = GetTextureMinMax(m_cached_ctx.TEX0, m_cached_ctx.CLAMP, m_vt.IsLinear(), false, m_vt.m_primclass == GS_SPRITE_CLASS && PrimitiveCoversWithoutGaps()).coverage;
```

Comment:
I think it's worth moving `PrimitiveCoversWithoutGaps()` into `GSState` (if it isn't already), and then moving this call into `GetTextureMinMax()`. It's a potentially costly operation (that's why I memoized the result), and we don't want to be doing it for all sprites unless all the preconditions for needing it are satisfied.

Reply:
I did think about doing that, yeah, I may very well do that.  I still only do it when it's FST and sprite and no gaps, there's no other scenario

Reply:
Maybe we should consider equal-q (or q=1).. IIRC the expanding case I was looking at the other day used normalized coordinates.

Reply:
we could do, yea, since the vertex trace has already worked out the texture dimensions